src/OrchardCore/Orchard.DisplayManagement.Liquid/Fluid/Tags/HelperTag.cs

```diff
@@ -117,7 +117,7 @@ public HelperStatement(ArgumentsExpression arguments, IList<Statement> statement
 
                 foreach (var attribute in descriptor.Attributes)
                 {
-                    if (attribute.IsNameMatch(attributeName))
+                    if (attribute.IsNameMatch(attributeName) || attribute.IsNameMatch("asp-" + attributeName))
```

Comment:
Yes, just did it yesterday for testing. I will change it.

> ... will just call the tag helper 's WriteToAsync to reuse some code ...

Here you mean the tag helper `Process() or ProcessAsync()`. Right?